src/templates/blog-post/blog-post.js

```diff
@@ -1,12 +1,11 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import { kebabCase } from 'lodash'
-import Helmet from 'react-helmet'
-import { graphql, Link } from 'gatsby'
-import Layout from '../../components/Layout'
-import Content, { HTMLContent } from '../../components/Content'
-import './blog-post.scss'
-
+import React from 'react';
+import PropTypes from 'prop-types';
+import { kebabCase } from 'lodash';
+import Helmet from 'react-helmet';
+import { graphql, Link } from 'gatsby';
+import Layout from '../../components/Layout';
+import Content, { HTMLContent } from '../../components/Content';
+import './blog-post.scss';
```

Comment:
would be nice to switch this scss out also

Reply:
Will do the css for all markdown into another PR